RoomsQuickstart/Program.cs

```diff
@@ -53,18 +60,10 @@ static async Task CreateRoom()
                 DateTimeOffset validFrom = DateTimeOffset.UtcNow;
                 DateTimeOffset validUntil = DateTimeOffset.UtcNow.AddDays(1);
 
-                if (roomsClient is not null)
-                {
-                    RoomModel createRoomResult = await roomsClient.CreateRoomAsync(validFrom, validUntil, RoomJoinPolicy.InviteOnly, roomParticipants, cancellationToken);
-                    rooms.Add(createRoomResult.Id);
-
-                    RoomModel roomInfo = await roomsClient.GetRoomAsync(createRoomResult.Id);
-                    PrintRoom(roomInfo);
-                }
-                else
-                {
-                    throw new Exception("roomsClient is null");
-                }
+                RoomModel createRoomResult = await roomsClient.CreateRoomAsync(validFrom, validUntil, RoomJoinPolicy.InviteOnly, roomParticipants, cancellationToken);
```

Comment:
can we use var createdRoom?

Reply:
By using RoomModel type, it implicitly converts the response into RoomModel class. otherwise, using var, it gives object of type response